src/analysis/retail/shaman/enhancement/modules/dragonflight/Tier30.tsx

```diff
@@ -99,31 +99,36 @@ export default class Tier30 extends Analyzer {
           <>
             <SpellLink spell={SPELLS.VOLCANIC_STRENGTH_TIER_BUFF} />
             <ul>
-              <li>
+              <li key="phys">
```

Comment:
just as a note: the `key` prop is only needed when using arrays as JSX. when you've spelled it out like this the `key` prop doesn't add anything. not something that needs fixing, but a note

Reply:
I'll remove them, I think I was a bit overzealous when trying to fix the missing key properties I just added it to every `<li>` I've used in the entire enhancement module